website/project/views/node.py

```diff
@@ -791,7 +792,7 @@ def _view_project(node, auth, primary=False,
         ]
     }
     if embed_contributors and not anonymous:
-        data['node']['contributors'] = utils.serialize_contributors(node.visible_contributors, node=node)
+        data['node']['contributors'] = utils.serialize_visible_contributors(node)
```

Comment:
If this wasn't for a mako view I'd say we should put this method on the model but since it's probably all going to be thrown away anyway, for either django views or ember views, I won't say anything.